terraform/aspen_batch_iam.tf

```diff
@@ -1,6 +1,65 @@
+data "aws_secretsmanager_secret" "gisaid-download-credentials" {
+  name = "gisaid-download-credentials"
+}
+
 resource "aws_iam_role" "aspen-batch-jobs-role" {
   name = "aspen-batch-jobs-role"
   assume_role_policy = templatefile("${path.module}/iam_templates/trust_policy.json", {
     trust_services = ["ecs-tasks"]
   })
 }
+
+
+resource "aws_iam_policy" "aspen-batch-jobs-policies" {
+  name = "aspen-batch-jobs-policies"
+  policy = <<-POLICY
```

Comment:
is this syntax correct or is it supposed to be `<<POLICY`? Why use this syntax instead of `<<EOF` (which seems to be what the terraform docs prefer)

Reply:
<<- allows you to indent the input.  see https://www.terraform.io/docs/language/expressions/strings.html#indented-heredocs

Reply:
oh sweet didn't know that